README.md

```diff
@@ -71,6 +71,189 @@ const result = validate(value, validations)
+const result = validate(value, validations) 
+/* {
+    value: 'hello',
+    errors: [{ notContains: "lorem ipsum dolor" }]
+} */
+
+
+var object = { foo: true }
+const value = 'foo'
+var options = { allowed: object }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'foo',
+    errors: []
+} */
+
+var object = { foo: true }
+const value = 'bar'
+var options = { allowed: object }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'bar',
+     errors: [{ notContains: { foo: true } }]
+} */
+
+```
+
+**notAllowed option examples:**
+
+```javascript
+var list = ["small", "medium", "large"]
+const value = 'xlarge'
+var options = { notAllowed: list }
+const validations = { contains: list }
+const result = validate(value, validations) 
+/* {
+    value: 'xlarge',
+    errors: []
+} */
+
+var list = ["small", "medium", "large"]
+const value = 'small'
+var options = { notAllowed: list }
+const validations = { contains: list }
+const result = validate(value, validations) 
+/* {
+    value: 'small',
+    errors: [{ contains: ["small", "medium", "large"] }]
+} */
+
+var text = "lorem ipsum dolor"
+const value = 'hello'
+var options = { notAllowed: text }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'hello',
+    errors: []
+} */
+
+var text = "hello world"
+const value = 'hello'
+var options = { notAllowed: text }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'hello',
+    errors: [{ contains: "hello world" }]
+} */
+
+
+var object = { foo: true }
+const value = 'bar'
+var options = { notAllowed: object }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'bar',
+    errors: []
+} */
+
+var object = { foo: true }
+const value = 'foo'
+var options = { notAllowed: object }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'foo',
+     errors: [{ contains: { foo: true } }]
+} */
+
+```
+
+**using both options examples:**
+
+```javascript
+var allowedList = ["small", "medium", "large"]
+var notAllowedList = ["xlarge", "xxlarge", "tiny"]
+const value = 'large'
+var options = { allowed:allowedList, notAllowed: notAllowedList }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'xlarge',
+    errors: []
+} */
+
+var allowedList = ["small", "medium", "large"]
+var notAllowedList = ["xlarge", "xxlarge", "tiny"]
+const value = 'regular'
+var options = { allowed:allowedList, notAllowed: notAllowedList }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'regular',
+    errors: [{ notContains: ["small", "medium", "large"] }]
+} */
+
+var allowedList = ["small", "medium", "large"]
```

Comment:
how is this different from the preview example?

Reply:
In the second example it shows a double error. The existence of a value when it shouldn't have and the non-existence of a value when it should.